src/lib.rs

```diff
@@ -67,3 +67,19 @@ unsafe extern "C" fn clock_gettime(
 
     retval
 }
+
+#[no_mangle]
+unsafe extern "C" fn getrandom(
+    buf: *mut libc::c_void,
+    buflen: libc::size_t,
+    _flags: libc::c_uint,
+) -> libc::ssize_t {
+    let ret = ctru_sys::psInit();
```

Comment:
Does this need to happen every time, or can we lazily initialize it with something like [`core::sync::Once`](https://doc.rust-lang.org/stable/std/sync/struct.Once.html)?

Also, do we need to call `psExit` or not worry about it?

Reply:
Good question re: `psExit`. The docs are pretty sparse on what these functions actually do, but I would assume that it is more correct to call it before the program ends. I'm not really sure how we could handle something like that in this crate... maybe this impl should only call `PS_GenerateRandomBytes` and it's up to the user to initialize / exit PS itself?

Reply:
Oh, also – as far as I can tell there's no problem calling it more than once, but it could be done with `Once` if we wanted to avoid the potential overhead

Reply:
If it's OK with not calling `psExit`, then I'd go with the `Once` route.

Reply:
I opted to remove `psInit` entirely, as I think it makes more sense to treat `Ps` the way e.g. `Gfx` and `Apt` are in `ctru-rs` already. With the new implementation I pushed, failing to initialize the `Ps` causes a runtime panic if the user tries to e.g. construct a `HashMap`, but everything works if they do initialize it.